app/javascript/gobierto_visualizations/webapp/visualizations/treeMapNested.vue

```diff
@@ -229,36 +237,19 @@ export default {
         const g1 = svg
           .append("g")
           .datum(d)
-          .attr('class', d => {
-            let parentNameContract = ''
-            const { depth } = d
-            if (depth === 1) {
-              const { data: { name = '' } } = d
-              parentNameContract = name
-            } else if (depth === 2) {
-              const { parent: { data: { name = '' } } } = d
-              parentNameContract = name
-            }
-            //Normalize and create an slug because Servicios de Gestión Públicos isn't a valid class
-            parentNameContract = normalizeString(parentNameContract)
-            return `depth ${parentNameContract}`
-          })
+          .attr('class', 'depth')
 
         const g = g1.selectAll(".children")
           .data(d.children)
           .join("g")
 
         g.filter(d => d.children)
           .join('rect')
-          .attr('class', d => {
-            //The name can be the contractor or the type of contract so we have to extract the names of the object, and its parent
-            const { data: { name }, parent: { data: { name: parentName = '' } } } = d
-            let parentNameContract = parentName
-            let sectionName = name
-            //Normalize and create an slug because Servicios de Gestión Públicos isn't a valid css class
-            sectionName = normalizeString(sectionName)
-            parentNameContract = normalizeString(parentNameContract)
-            return `children ${sectionName} ${parentNameContract}`
+          .attr('class', 'children')
+          .attr('fill', d => {
+            const { depth, data: { name = '' }, parent: { data: { name: parentName = '' } } } = d
+            const valueColor = depth === 1 ? d.color = colors(name) : d.color = colors(parentName)
```

Comment:
Mmmm, you're setting `d.color`, but you return `valueColor`. Is that any sense?

Reply:
Done!

Reply:
Wait there. Why are you doing an assignment `d.color = colors(name)` or `d.color = colors(parentName)` inside the returning value (or the previous `valueColor`)? That's what I wanted to pointed out.